modules/alloy/src/test/ts/browser/position/SelectionInDocPositionTest.ts

```diff
@@ -133,6 +154,47 @@ UnitTest.asynctest('SelectionInDocPositionTest', (success, failure) => {
           PositionTestUtils.cTestSink(
             'Fixed, Selected: 13rd paragraph, large scroll, no editor scroll',
             'fixed'
+          ),
+
+          ChainUtils.cLogging(
+            'Setting selection to 5th-7th paragraph and scrolling there',
+            [
+              NamedChain.writeValue('path', Cursors.path({
+                startPath: [ 5 ],
+                soffset: 0,
+                finishPath: [ 8 ],
+                foffset: 0
+              })),
+              NamedChain.bundle((data: any) => {
+                const root = data.inline.element;
+                // Scroll so the 7th paragraph is at the top
+                const range = Cursors.calculate(root, Cursors.path({
+                  startPath: [ 7 ],
+                  soffset: 0,
+                  finishPath: [ 8 ],
+                  foffset: 0
+                }));
+                Scroll.intoView(range.start, true);
+                return Result.value(data);
+              }),
+
+              // Update the anchor
+              NamedChain.write('anchor', cSetupAnchor)
+            ]
+          ),
+
+          PositionTestUtils.cTestSink(
+            'Relative, Selected: 5th-7th paragraph, no scroll, some editor scroll',
+            'relative'
+          ),
+          PositionTestUtils.cTestSink(
+            'Fixed, Selected: 5th-7th paragraph, no scroll, some editor scroll',
+            'fixed'
+          ),
+
+          ChainUtils.cLogging(
```

Comment:
This only tests the fixed position sink, it needs to be done after each `cTestSink` above.

- [ ] Ensure the position is tested when positioning in both relative and fixed position sinks